trio/_subprocess/tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
```

Comment:
Instead of making another copy of this, I'm thinking we should put the subprocess tests into `trio/tests/subprocess/`? Or `_subprocess/` or `test_subprocess/`, I guess, whichever looks nicest. (And at some point we should move the `_core` tests into `trio/tests/` too. And probably rename it to `trio/_tests/`. But that can wait for another PR...)